### ISSUE TITLE
EditorScript _edit() method can receive null objects but the signature in gdext is Gd<Object> instead of Option<Gd<Object>>

### ISSUE THREAD

**Issue by lmsonic** [godot-rust/gdext#880]
The method [_edit(object:Object)](https://docs.godotengine.org/en/stable/classes/class_editorplugin.html#class-editorplugin-private-method-edit) in EditorScript can accept null values, but the API in gdext accepts Gd\<Object\> which cannot be null
This is my usage right now, but it gives me errors in any case on the godot side:
```rust
fn edit(&mut self, object: Gd<Object>) {
        if !object.is_instance_valid() {
            return;
        }
        if let Ok(curve) = object.try_cast() {
            self.curve = Some(curve);
        }
    }
```
Error:
```
C:\Users\<REDACTED>\.cargo\registry\src\index.crates.io-6f17d22bba15001f\godot-core-0.1.3\src\private.rs:313 - godot-rust function call failed: SolidCurve2DEditorPlugin::edit()
      Reason: [panic]  in function `SolidCurve2DEditorPlugin::edit` at parameter [0] of type godot_core::obj::gd::Gd<godot_core::gen::classes::object::re_export::Object>: `Gd` cannot be null: null
```

**Comment by Bromeon** [comment]
Please search for existing issues before opening new ones (e.g. `edit` yields it):
https://github.com/godot-rust/gdext/issues/494

**Comment by Bromeon** [comment]
Now fixed by #883.
